If subshells are supported by the kernel, send comm messages to subshells (#17363)
jupyterlab/jupyterlab: CHANGELOG.md

<img alt="The subshell console" src="https://raw.githubusercontent.com/jupyterlab/jupyterlab/main/docs/source/getting_started/changelog_assets/4.4-subshells.png" class="jp-screenshot">

> Alternatively, JupyterLab allows automatically creating subshells for comm messages if kernels support this feature.
> This has the by-product of allowing to execute `ipywidgets` callbacks concurrently without requiring a user action.
> This behavior can be changed from the settings, the user can either:

Reviewer: ```suggestion
JupyterLab can automatically create subshells for comm messages if kernels support this feature.
This enables concurrent execution of `ipywidgets` callbacks without requiring a user action.
This behavior can be changed from the settings, with options to:
```
